Pipfile

```diff
@@ -10,4 +10,4 @@ platformshconfig = {editable = true,path = "."}
 pytest = "*"
 
 [requires]
-python_version = "3.6"
+python_version = "3.7"
```

Comment:
If this bumps the version requirement, that's technically an API change so would require a new major.  I'd rather not go for a new major for this one library yet unless we really have to.